cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/strategy/PostponeUrpReplicaMovementStrategy.java

```diff
@@ -15,11 +15,16 @@
  */
 public class PostponeUrpReplicaMovementStrategy extends AbstractReplicaMovementStrategy {
 
+  @Override
+  public Comparator<ExecutionTask> taskComparator(StrategyOptions strategyOptions) {
+    return (task1, task2) -> isPartitionUnderReplicated(strategyOptions.cluster(), task1.proposal().topicPartition())
+                             ? (isPartitionUnderReplicated(strategyOptions.cluster(), task2.proposal().topicPartition()) ? 0 : 1)
```

Comment:
```suggestion
                             ? (isPartitionUnderReplicated(strategyOptions.cluster(), task2.proposal().topicPartition()) ? PRIORITIZE_NONE : PRIORITIZE_TASK_2)
```
I like your `int PRIORITIZE_TASK_2 = 1` a lot. Is refactoring them here a good idea?

Reply:
Sure -- applied the proposed change.